src/runtime_src/core/include/xrt.h

```diff
@@ -335,9 +335,9 @@ xclGetErrorStatus(xclDeviceHandle handle, struct xclErrorStatus *info);
  * @buffer:        Pointer to device image (xclbin) in memory
  * Return:         0 on success or appropriate error number
  *
- * Download FPGA image (AXLF) to the device. The PR bitstream is
- * encapsulated inside xclbin as a section. xclbin may also contains
- * other sections which are suitably handled by the driver.
+ * | Download FPGA image (AXLF) to the device. The PR bitstream is encapsulated inside xclbin as a section. 
+ * | xclbin may also contains other sections which are suitably handled by the driver.
+ * | **NOTE**: This API also downloads OVERLAY (dtbo) section. (Applicable only for Edge Platforms)
```

Comment:
Sorry, the lines are too long.  This is a public exported header files, it must be formatted properly.  What's the deal with the '|' character?   I know the original comment didn't read well either but at least we should fix that.

```suggestion
 * Download FPGA image (AXLF) to the device. The PR bitstream is
 * encapsulated inside xclbin as a section.  The xclbin may also
 * contain other sections, which are suitably handled by the driver.
 *
 * This API also downloads OVERLAY (dtbo) section (Edge only).
```

Reply:
Hi @stsoe , I am trying to get NOTE in the new line. '|' is used for new line. With the suggested change its not possible, so I sent the commit this way. I will do some more trials and update

Reply:
Please just commit my change.  I don't see the need for a NOTE.

Reply:
Thanks soren, I will commit the changes as suggested